.github/workflows/ci.yml

```diff
@@ -445,6 +445,8 @@ jobs:
           # By default, `libparsec_crypto` uses Rust Crypto
           cargo test --profile ci-rust --package libparsec_crypto --features use-sodiumoxide
         timeout-minutes: 30
+        env:
+          RUST_LOG: trace
```

Comment:
Should we use the trace level ? Won't it produce too much noise ?

Reply:
I set debug then